### PR TITLE
AILab: Use published ml-playground app

### PR DESCRIPTION
**Pull request by breville**
This sets up the AI Lab app to be embedded in https://studio.code.org.  That app is in https://github.com/code-dot-org/ml-playground.

An example level is available at https://studio.code.org/s/allthethings/stage/43/puzzle/1.

See https://github.com/code-dot-org/ml-playground/pull/25 for the initial publish of the app.

This was mostly based on what was done for Oceans.

### CODE REVIEW

File: apps/src/ailab/Ailab.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ Ailab.prototype.init = function(config) {
   config.pinWorkspaceToBottom = true;
 
   const onMount = () => {
-    // NOTE: Other apps call studioApp.init(), except WebLab. Fish is imitating WebLab
+    // NOTE: Other apps call studioApp.init(), except WebLab. Ailab is imitating WebLab.
```
Comment (Erin007) [inline apps/src/ailab/Ailab.js]: So, is it now true that all other apps, except Fish AND Weblab AND AiLab, call studioApp.init()?

File: apps/src/ailab/Ailab.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ Ailab.prototype.injectStudioApp = function(studioApp) {
  */
 Ailab.prototype.init = function(config) {
```
Comment (Erin007) [inline apps/src/ailab/Ailab.js:42]: This was already called AiLab in Fish?

File: apps/src/ailab/Ailab.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+  ReactDOM.render(
+    <Provider store={getStore()}>
+      <AilabView onMount={onMount} />
+    </Provider>,
+    document.getElementById(config.containerId)
+  );
+};
+
+// Called by the fish app when it wants to go to the next level.
```
Comment (Erin007) [inline apps/src/ailab/Ailab.js]: Is this comment about fish still relevant?

**Review by Erin007 — APPROVED** [review]
Ahh! This is so cool! I wonder if it's worth adding a UI test for the allthethings level just to make sure something actually renders? And might be good to get 👀  from someone who's made a new App type before just to be safe. Otherwise, LTGM!

**Comment by breville** [comment]
Good call, but I think I will hold off on UI/Eyes tests until we get a bit further on UI.  Even the responsive solution here, copied from Fish, might not be the right one for this new app.

**Pull request merged**